app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/model/SourceDefinition.scala

```diff
@@ -44,11 +44,11 @@ object SourceDefinition extends LazyLogging {
   ) {
     @SuppressWarnings(Array("OptionGet"))
     def toSourceDefinition: SourceDefinition = {
-      if (extent.isDefined && crs.isDefined && cellSize.isDefined) {
+      if (extent.isDefined && crs.isDefined && cellSize.isDefined && extentCrs.isDefined) {
```

Comment:
The one thing the current implementation preserves is if there are partial parameters like 3 of the 4 are defined the one supplied will take precedence. I added the case statement, but maybe it should always be get or else.

I'm also of half the mind that it should just be using the tiff tags and that should be our source of truth.